docs/en/enterprise-edition/policy-reference/aws-policies/aws-networking-policies/bc-aws-375.adoc

```diff
@@ -0,0 +1,56 @@
+
+== AWS S3 bucket has global view ACL permissions enabled
+
+=== Policy Details
+
+[width=45%]
+[cols="1,1"]
+|===
+|Prisma Cloud Policy ID
+| 43c42760-5283-4bc4-ac43-a80e58c4139f
+
+|Checkov ID
+| https://github.com/bridgecrewio/checkov/blob/main/checkov/terraform/checks/resource/aws/S3GlobalViewACL.py[CKV_AWS_375]
+
+|Severity
+|LOW
+
+|Subtype
+|Run,Build
+
+|Frameworks
+|Terraform,TerraformPlan
+
+|===
+
+=== Description
+
+This policy is checking to ensure that an AWS S3 bucket does not have global view ACL (Access Control List) permissions enabled. The focus is on preventing the bucket from being publicly accessible by anyone on the internet, which could happen if global read permissions are allowed through its ACL settings.
+
+Having global view permissions enabled on an S3 bucket means that anyone with the correct URL can view the contents of the bucket, potentially exposing sensitive data or files. This is a security risk because it can lead to unauthorized access or data breaches, wherein malicious actors could exploit the publicly available data for nefarious purposes. Ensuring that S3 buckets do not have these permissions enabled is crucial for maintaining data privacy and security in cloud environments.
+
+=== Fix - Buildtime
+
+*Terraform*
+
+* *Resource:* aws_s3_bucket_acl
+* *Arguments:* access_control_policy
+
+Ensure that your AWS S3 bucket does not have global view permissions by avoiding 'public-read', 'public-read-write', or 'authenticated-read' ACL settings. Properly restrict access by setting the ACL to 'private' or using more specific bucket policies and IAM roles.
+
+Here's how to update your Terraform configuration to ensure the S3 bucket does not have global view ACL permissions:
```

Comment:
This example demonstrates how to modify your Terraform configuration to ensure the S3 bucket's ACL does not have global view ACL permissions.